src/components/NavBar.js

```diff
@@ -51,9 +55,13 @@ class NavBar extends React.PureComponent {
           <img
             className="logo"
             src={Logo}
-            alt="People Power Solar Cooperative Logo"
+            alt="North Shore Economic Vitally Partnership Logo"
           />
         </a>
+        <div style={{ width: '100%', marginLeft: '16px' }}>
+          <h3 style={{ margin: '0px' }}>North Shore</h3>
+          <h3 style={{ margin: '0px' }}>EVP</h3>
```

Comment:
Change text on the navbar tabs to: dashboard, farms, supply
Add new farm button
Replace gear with profile picture